tests/test_big_modeling.py

```diff
@@ -338,6 +338,24 @@ def test_dispatch_model_with_non_persistent_buffers(self):
 
         with TemporaryDirectory() as tmp_dir:
             dispatch_model(model, device_map, offload_dir=tmp_dir, offload_buffers=True)
+
+            output = model(x)
+            self.assertTrue(torch.allclose(expected, output.cpu(), atol=1e-5))
+
+    def test_dispatch_model_with_cpu_offload(self):
+        model = ModelForTest()
+        device_map = {"linear1": "disk", "batchnorm": "cpu", "linear2": 0}
+
+        x = torch.randn(2, 3)
+        expected = model(x)
+
+        with TemporaryDirectory() as tmp_dir:
+            dispatch_model(model, device_map, offload_dir=tmp_dir, cpu_offload=True)
+
+            self.assertEqual(model.linear1.weight.device, torch.device("meta"))
+            self.assertEqual(model.batchnorm.weight.device, torch.device("cpu"))
```

Comment:
The new behavior of getting "cpu" here instead of "meta" looks more intuitive to me.

Reply:
Yes this is what we are aiming in this PR ! We want to let the module on `cpu` and not on `meta` device.